dipy/segment/tests/test_feature.py

```diff
@@ -284,6 +285,26 @@ def extract(self, streamline):
     d2 = metric.dist(features1, features2)
     assert_equal(d1, d2)
 
+    class ArcLengthFeature(dipymetric.Feature):
+        def infer_shape(self, streamline):
+            if sys.version_info > (3,):
```

Comment:
Oh - sorry - I see you are trying to test the Windows behavior on non-Windows machines - maybe just a comment to that effect.